src/plone/volto/browser/migrate_to_volto.pt

```diff
@@ -13,30 +13,34 @@
     <p class="lead" i18n:translate="">Here you can prepare this site for Volto.</p>
 
     <h3 i18n:translate="">What is Volto?</h3>
-    <p i18n:translate="">Volto is a React-based frontend for Plone. It is the default UI for Plone 6.</p>
+    <p i18n:translate="">Volto is a React-based frontend for Plone. It is the default UI for Plone 6. Volto is a separate server-side application.</p>
+
+    <h3 i18n:translate="">What do you need for Volto?</h3>
+    <p i18n:translate="">Volto needs to run continuously alongside the current Plone backend service from which you access this wizard. The Volto 
+                         server performs initial server-side rendering (SSR) of the React application for pages. It also delivers the JavaScript frontend
+                         to your browser. Without a Volto frontend prepared for this website and the means to host it, continuing this migration makes no
+                         sense! Read the <a href="https://6.docs.plone.org/volto/index.html">Volto Frontend documentation</a> for more information.</p>
 
     <h3 i18n:translate="">What will happen?</h3>
-    <p i18n:translate="">When you click on "Migrate to Volto" the following things will happen:</p>
+    <p i18n:translate="">When you click on "Migrate to Volto" at the end of the form, the following things will happen:</p>
     <ol>
         <li i18n:translate="">Install the package <code>plone.volto</code> and <code>plone.restapi</code>.</li>
-        <li i18n:translate="">The html of Richtext-Fields (previously edited using TinyMCE) is converted into Volto blocks so you can edit it in Volto.</li>
-        <li i18n:translate="">Pages, News Items and Events are made folderish. That means they can contain other content like Images or Pages.</li>
-        <li i18n:translate="">Default Pages of Folders are merged with the Folderish Pages that replace the Folder whereever that is possible.
-            This works well with Pages and Collections where the text and/or query are added to the folderish page that repalces the Folder.</li>
+        <li i18n:translate="">The HTML of Richtext-Fields (previously edited using TinyMCE) is converted into Volto blocks so you can edit it in Volto.</li>
```

Comment:
I missed one on the first pass.

```suggestion
        <li i18n:translate="">The HTML of Richtext fields (previously edited using TinyMCE) is converted into Volto blocks so you can edit it in Volto.</li>
```

Reply:
Lets go for RichText then. This is also the spelling in our Python code.  We could circumvent the issue with 'styled text', but images could be part of a RichText but not a styled text.

Reply:
I'll also update our spell checker. I've been fiddling with Vale, and I am impressed.